distsql/xeval/eval_compare_ops.go

```diff
@@ -231,8 +230,8 @@ func containsAlphabet(s string) bool {
 }
 
 func (e *Evaluator) evalIn(expr *tipb.Expr) (types.Datum, error) {
-	if len(expr.Children) != 2 {
-		return types.Datum{}, ErrInvalid.Gen("IN need 2 operands, got %d", len(expr.Children))
+	if len(expr.Children) < 1 {
```

Comment:
Since "IN needs more than 1 operand", should it be `if len(expr.Children) < 2 {` ?